_health-care/_js/_form.js

```diff
@@ -1,21 +1,47 @@
 var HealthApp = (function() {
   "use strict";
 
-  var identityTransform = function(formData, arg) { return arg; };
-  var extractTransform = function(formData, arg) { return formData[arg]; };
+  function identityTransform(formData, arg) { return arg; }
+  function extractTransform(formData, arg) { return formData[arg]; }
```

Comment:
...though now that I've written that, I actually realized I don't know where I got this idea from. I will look at that later today to confirm.